model/model.go

```diff
@@ -748,6 +752,7 @@ type PartitionInfo struct {
 	AddingDefinitions []PartitionDefinition `json:"adding_definitions"`
 	// DroppingDefinitions is filled when dropping a partition that is in the mid state.
 	DroppingDefinitions []PartitionDefinition `json:"dropping_definitions"`
+	States              []PartitionState      `json:"states"`
```

Comment:
should keep the states element offset be consistent with the definition?

Reply:
I think this is unnecessary.